### ISSUE TITLE
ProtonQt closes itself after 1 minute

### ISSUE THREAD

**Issue by Ondra3211** [DavidoTek/ProtonUp-Qt#44]
I am running the flatpak version and the  app closes it self exactly after 1 minute. No errors in terminal.

flatpak: ProtonUp-Qt v2.6.0
Arch, KDE Plasma 5.24.3

**Comment by DavidoTek** [comment]
Does it happen when you are not using the app or does it also happen when you install a compatibility tool?

Is there a gamepad connected to your computer?

**Comment by Ondra3211** [comment]
I managed to fix it. On first run there is a popup window to allow PorotonQt run in background. If it's not accepted it will crash after few second with the 137 error code.
![Screenshot_20220322_222755](https://user-images.githubusercontent.com/39314277/160252166-c2818974-12ec-4bd3-938b-1b4f52b9f436.png)
So I just run 
```
flatpak permission-reset net.davidotek.pupgui2
flatpak run net.davidotek.pupgui2
```

After resseting permissions the popup window showed again and I accepted it.
